hack/common-vars.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ROOTDIR=$(dirname "${BASH_SOURCE[0]}")/..
+# shellcheck disable=SC2034
+KUBECTL="${ROOTDIR}/hack/tools/bin/kubectl"
+# shellcheck disable=SC2034
+KUSTOMIZE="${ROOTDIR}/hack/tools/bin/kustomize"
+# shellcheck disable=SC2034
```

Comment:
I feel ok with disabling the shellcheck error using `# shellcheck disable=SC2034`. 
`Export`ing the variable also solves the shellcheck error, but that can cause multiple exports of the same variable in a hypothetical scenario where `common-vars.sh` gets referenced multiple times in subprocesses.

Reply:
why is this triggering shellcheck?